rosdep/python.yaml

```
@@ -361,6 +361,14 @@ pika:
     pip:
       packages: [pika]
   ubuntu: [python-pika]
+platformio:
+  debian:
+    pip:
+      packages: [platformio]
+  fedora: [python-platformio]
```

```suggestion
  fedora: [platformio]
```

(on Fedora, it looks like the `platformio` package provides the command-line utils, equivalent to what `platformio` does on Ubuntu)

---

It seems so.
I have made the corrections.